### DISCUSSION TITLE
"404 Account Not Found" error keeps showing up

### DISCUSSION THREAD

**Discussion by mohamedaleya** [emailjs-com/help#43] [Q&A] [answered]
I cannot send test or even real emails because it keeps saying 404 Account Not Found, even though the account exists, and I have used it before with EmailJS.

I'm using Gmail however I've also tried it with Outlook yet it's the same thing.
I'm using this with React, however it still doesn't work even from the browser so it's not framework or setup related.

![DinuVcJZb0](https://github.com/emailjs-com/help/assets/43639707/6adc9807-2b70-4f43-a7b0-4f80d25d9c96)

EDIT: As I am writing this, everything is back to normal suddenly. Kept this in case someone else had the same issue. I'm guessing the API went partially down(?)

**Comment by xr0master** [comment, marked as answer]
This means that the public key is not correct because the account associated with the key was not found.
